File: 75-sort-colors/sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
    const red = nums.filter(num => num === 0).length;
    const white = nums.filter(num => num === 1).length;
    const blue = nums.filter(num => num === 2).length;

    const sorted = Array(red).fill(0).concat(Array(white)
    .fill(1)).concat(Array(blue).fill(2));

    for (let i = 0; i < nums.length; i++) {
        nums[i] = sorted[i];
    }
};

File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        red = nums.count(0)
        white = nums.count(1)
        blue = nums.count(2)

        x = [0] * red
        y = [1] * white
        z = [2] * blue

        sorted_nums = x+y+z
        for i in range(len(sorted_nums)):
            nums[i] = sorted_nums[i]